.coveragerc

```diff
@@ -1,7 +1,11 @@
 # .coveragerc to control coverage.py
 [run]
+branch = True
 omit =
-    src/scenic/simulators/webots/WBT*.py
+    tests/*
+    */__init__.py
```

Comment:
What's the reason for excluding the `__init__.py` files? If they are empty, I would think they would have 100% coverage anyway, and if they aren't empty we should probably compute coverage for them, since they do get run when Scenic is used.

Reply:
I just checked many of the `__init__.py` files and they were either empty or had some comments. Many had zero lines of coverage to actually report. And yes they would generate 100% coverage, but it would actually skew the average coverage percentage generated across all files. I am happy to re-add as needed!

Reply:
Showing skew for future runs and leaving in `__init__.py`.